### PR TITLE
DCGAN_pytorch

### PR DESCRIPTION
**Pull request by nakul-jindal**

### CODE REVIEW

File: generative_models/DCGAN_pytorch/main.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+dataloader = DataLoader(dataset, batch_size=batch_size, shuffle=True, num_workers=2)
+
+from models import Generator , Discriminator , weights_init
+netD = Discriminator(channels_img, features_d).to(device)
+netG = Generator(channels_noise, channels_img, features_g).to(device)
+netG=netG.apply(weights_init)
+netD=netD.apply(weights_init)
+
+optimizerD = optim.Adam(netD.parameters(), lr=lr, betas=(0.5, 0.999) )
+optimizerG = optim.Adam(netG.parameters(), lr=lr, betas=(0.5, 0.999) )
```
Comment (nirmal-suthar) [inline generative_models/DCGAN_pytorch/main.py]: use beta1 args

File: generative_models/DCGAN_pytorch/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# DCGAN implementation in pytorch on MNIST
+
+contributed by : [Nakul Jindal](https://github.com/nakul-jindal)
+
+original paper :  [Unsupervised Representation Learning with Deep Convolutional Generative Adversarial Networks](https://arxiv.org/abs/1511.06434)
+
+MNIST dataset : http://yann.lecun.com/exdb/mnist/
```
Comment (nirmal-suthar) [inline generative_models/DCGAN_pytorch/README.md]: Can you reformat it similar to other implementation (add usage, help log, contributed by, and reference)

File: generative_models/DCGAN_pytorch/README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
```
Comment (nirmal-suthar) [inline generative_models/DCGAN_pytorch/README.md:45]:
```suggestion
# Summary
```

**Review by nirmal-suthar — APPROVED** [review]

File: generative_models/DCGAN_pytorch/main.py (hunk 1 above)
Comment (nakul-jindal) [reply generative_models/DCGAN_pytorch/main.py:47]: didnt understand , what about beta1 args?

Comment (nakul-jindal) [reply generative_models/DCGAN_pytorch/main.py]: ok got it

Comment (nakul-jindal) [reply generative_models/DCGAN_pytorch/main.py]: dekhlo sare changes bana diye

Comment (nakul-jindal) [reply generative_models/DCGAN_pytorch/main.py]: also i had picked wGAN but i am submitting dcgan because i was new to gans so couldnt get the hang of it

**Pull request merged**